adapters/beachfront/beachfront.go

```diff
@@ -550,18 +558,22 @@ func (a *BeachfrontAdapter) getBidType(externalRequest *adapters.RequestData) op
 
 func postprocess(response *adapters.ResponseData, xtrnal openrtb.BidRequest, uri string, id string) ([]openrtb.Bid, []error) {
 	var beachfrontResp []beachfrontResponseSlot
-	var errs = make([]error, 0)
 
 	var openrtbResp openrtb.BidResponse
 
 	// try it as a video
-	if err := json.Unmarshal(response.Body, &openrtbResp); err != nil {
-		errs = append(errs, err)
+	if err := json.Unmarshal(response.Body, &openrtbResp); err != nil || len(openrtbResp.SeatBid) == 0 {
+		// when running postman tests with the request in supplemental/bidder_response_unmarshal_error_{adm|nurl}_video.json, an
+		// error was thrown here by the attempt at un-marshaling, and, correctly, the true branch was followed. When running
+		// those tests through ./validate.sh, the same request failed the condition, err was nil, and the false branch was
+		// followed, resulting in an "index out of range" error at the postprocessVideo call below. I am not clear on how that
+		// passed, but I added the "or" clause to cover it. That works, though it looks redundant.
```

Comment:
If you copied the request verbatim from `supplemental/bidder_response_unmarshal_error_{adm|nurl}_video.json` and sent it via postman, you should get different results from running `./validate.sh`. The request in those JSON files contain the following in `mockBidRequest`:
```
"imp": [
      {
        "id": "video1",
        "ext": {
          "bidder": {
            "bidfloor": 3.01,
            "appId": "videoAppId1"
          }
```

The key `bidder` should be `beachfront` if you're sending via postman since there is some upstream processing taking place under normal conditions that aren't happening when you run the JSON tests.

Reply:
I see.
Thanks.
I'll go ahead and remove the comment.